terraform/overrides/nodes_use_ephemeral_storage_override.tf

```diff
@@ -0,0 +1,30 @@
+resource "openstack_compute_instance_v2" "node_server" {
+  block_device {
+    boot_index            = 0
+    delete_on_termination = true
+    destination_type      = "local"
+    source_type           = "image"
+    uuid                  = data.openstack_images_image_v2.image_node.id
+  }
+
+  block_device {
+    boot_index            = -1
+    delete_on_termination = true
+    destination_type      = "local"
+    source_type           = "blank"
+    volume_size           = var.volume_size_storage
```

Comment:
This needs some docs added, it is necessary to use a flavor that allows ephemeral_gb to have this size.

Maybe also add a test job for this whole thing?

Reply:
I am adding to the documentation in a 2nd PR and also adding a simple test for it there.